app/Services/StandService.php

```diff
@@ -46,7 +48,13 @@ class StandService
      */
     private const ASSIGN_STAND_MINUTES_BEFORE = 15.0;
 
-    private $allStands = [];
+    /**
+     * The maximum distance in meters from an airfields centre that we bother
+     * checking if an aircraft is sat on a stand. This is about 2.5 nautical miles.
+     */
+    private const DISTANCE_FROM_AIRFIELD_TO_CHECK_STANDS = 5000;
+
+    private $allStandsByAirfield = [];
```

Comment:
This is declared as an array, but a Collection is assigned to it in `getAllStandsByAirfield`

Reply:
Fixed!